apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -588,7 +588,12 @@ func (m *ModelManagerAPI) DestroyModels(args params.Entities) (params.ErrorResul
 		if err := m.authCheck(model.Owner()); err != nil {
 			return errors.Trace(err)
 		}
-		return errors.Trace(common.DestroyModel(m.state, tag))
+		st, releaser, err := m.pool.Get(tag.Id())
```

Comment:
yes, more or less - I made it so that common.DestroyModel expects you to pass through the model-specific State